### PR TITLE
Folders 154061511

### PR DESCRIPTION
**Pull request by jjdelc**

### CODE REVIEW

File: source/feature-guide/feature-folders.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
+Variable folders
+----------------
+
+Variable folders provide a new and faster way to organize
```
Comment (dianaclarke) [inline source/feature-guide/feature-folders.rst]: organize... what?

Comment (jjdelc) [reply source/feature-guide/feature-folders.rst]: :P

Comment (jjdelc) [reply source/feature-guide/feature-folders.rst]: https://github.com/Crunch-io/apidocs-sphinx/commit/731fc50d0c54e02534b485802b87a19348ebd6b0

File: source/endpoint-reference/endpoint-folders.rst [hunk 2]
```diff
@@ -8,6 +10,17 @@ Much like the variable hierarchical order, folders allow you to have
 nested folders containing dataset variables. Unlike the hierarchical
 order, a variable can only exist in one folder at the time.
 
+Basic concepts
+~~~~~~~~~~~~~~
+
+There are a number of system folders that exist with every dataset created.
+These system folders cannot be renamed, moved or deleted.
+
+* **Root folder**: The root folder is the top level folder where all other subfolders will be created by dataset editors. All the variables and subfolders here will be public universal for all users with access to the dataset. It is only possible to place public varibles under this tree.
```
Comment (digitalresistor) [inline source/endpoint-reference/endpoint-folders.rst]: s/varibles/variables/

Comment (digitalresistor) [reply source/endpoint-reference/endpoint-folders.rst]: s/public universal/publicly accessible/

**Review by digitalresistor — CHANGES_REQUESTED** [review]

File: source/endpoint-reference/endpoint-folders.rst (hunk 2 above)
Comment (jjdelc) [reply source/endpoint-reference/endpoint-folders.rst]: Done, done!

**Review by digitalresistor — APPROVED** [review]

**Pull request merged**